### DISCUSSION TITLE
How to use pac auth to create a backup environment?

### DISCUSSION THREAD

**Discussion by AdrienDS78** [microsoft/powerplatform-actions#351] [Q&A]
Hello Team,

I would like to create a power platform environment backup with an application user but It seems to be created with my account as I run the command lines.

I don't understand how the authentication work with the appId & clientSecret and then if it is used by the following command line : 'pac admin backup  --environment xxxx    --label xxxx'

please help

**Comment by petrochuk** [comment]
you can create auth profile with appId & clientSecret. After that make sure this profile is selected and run pac admin backup
